tests/recommedSong.test.js

```diff
@@ -0,0 +1,31 @@
+const Song = require('../lib/song')
+const Playlist = require('../lib/playlist')
+const recommendSong = require('../lib/functions')
+
+
+test('Create a playlist and two separe songs to see if they are a good recommendation', () => {
+	let songs = [];
+    let song1 = new Song("Feo Fuerte y Formal", "Loquillo Y Los Trogloditas", "Rock");
+    let song2 = new Song("Venezia", "Hombres G", "Pop-Rock");
+    let collab3 = ["Loquillo Y Los Trogloditas", "Fito y Fitipaldis"];
+    let song3 = new Song("Luché contra la ley", collab3, "Rock");
+
+    songs.push(song1, song2, song3);
+
+    let suggestedSong1 = new Song("Antes de que cuente diez", "Fito y Fitipaldis", "Rock");
+    let suggestedSong2 = new Song("Good Vibrations - Remastered 2001", "The Beach Boys", "Pop");
+    let suggestedSong3 = new Song("Limbo", "Daddy Yankee", "Reggaeton");
+    let suggestedSong4 = new Song("She's Not There", "The Zombies", "Pop-Rock");
+
+    let suggestedSongs = [suggestedSong1, suggestedSong2, suggestedSong3, suggestedSong4];
+
+    let playlist = new Playlist(songs);
+
+    let recommended = recommendSong(playlist, suggestedSongs);
```

Comment:
De verdad, Carlos, dadle una vuelta a las cosas que hacéis, porque si no estáis perdiendo el tiempo. ¿De veras que para sugerir una canción se pasa una lista de canciones sugeridas y se te dice sí o no?
Por esto os digo qeu tenéis que plantear el problema. Si ponéis la solución delante del problema, es imposible que lo resolváis.

Reply:
No es eso, esa lista que se le pasa representa la discografía que tiene la aplicación. Como estamos hablando de que  es para hacer un test, la lista de la que se obtienen las recomendaciones es corta, pero es por eso mismo, porque es para un test y comprobar que funciona correctamente.

Reply:
No, no es para un test, es para la historia de usuario. Y una vez más, el API es importante. Para empezar, tienes que ver cuál es la discografía, que incluya o no todas las canciones. Y para eso tienes que tener un concepto claro de discografía y un modelo, no un simple argumento a una función. Y para eso tienes que modelizar claramente el problema.